integration-cli/events_utils.go

```diff
@@ -130,7 +130,7 @@ func matchEventLine(id, eventType string, actions map[string]chan bool) eventMat
 
 		if matchIDAndEventType(matches, id, eventType) {
 			if ch, ok := actions[matches["action"]]; ok {
-				close(ch)
+				ch <- true
```

Comment:
To avoid closing closed channel, It's better to use
<pre><code>select {
    	case <-ch:
    	default:
    	   close(ch)
    }</code></pre>  WDYT?